citrination_client/data/routes.py

```diff
@@ -0,0 +1,34 @@
+from citrination_client.util.quote_finder import quote
+
+def update_file(file_id):
+    return 'data_sets/update_file/{}'.format(file_id)
+
+def list_files(dataset_id):
+    return 'datasets/{}/list_filepaths'.format(dataset_id)
+
+def matched_files(dataset_id):
+    return 'datasets/{}/download_files'.format(dataset_id)
+
+def pif_dataset_uid(dataset_id, pif_uid):
+    return 'datasets/{}/pif/{}'.format(dataset_id, pif_uid)
+
+def pif_dataset_version_uid(dataset_id, version, pif_uid):
+    return 'datasets/{}/version/{}/pif/{}'.format(dataset_id, version, pif_uid)
+
+def create_dataset():
+    return 'data_sets/create_dataset'
```

Comment:
This is `data_sets` here but `datasets` in other routes. Is that correct?

Reply:
It is :(